examples/Echo.Server/EchoServerSettings.cs

```diff
@@ -8,11 +8,7 @@ public static class EchoServerSettings
     {
         public static bool IsSsl
         {
-            get
-            {
-                string ssl = ConfigurationManager.AppSettings["ssl"];
-                return !string.IsNullOrEmpty(ssl) && bool.Parse(ssl);
-            }
+            get { return ConfigurationManager.AppSettings["ssl"].ToLower() == "true"; }
```

Comment:
This will throw a `NullReferenceException` if `ssl` does not exist in `App.config`. The previous implementation did not.